EOSIO/eos
# Add trace plugin  API test

## Change Description


Testing:
1. Start Nodeos and Keosd
2. send transaction
3. get transaction id and block number
4. call trace API to get block, and verify transaction is in the block data

Jira: https://blockone.atlassian.net/browse/EPE-104

## Change Type
**Select ONE**

- [ ] Stability bug fix


- [ ] Other - special case




## Consensus Changes
- [ ] Consensus Changes




## API Changes




## Documentation Additions
- [ ] Documentation Additions
